cmd/apps/openfaas_app_test.go

```diff
@@ -22,3 +23,77 @@ func Test_getValuesSuffix_aarch64(t *testing.T) {
 		t.Errorf("suffix, want: %s, got: %s", want, got)
 	}
 }
+
+func Test_mergeFlags(t *testing.T) {
+	tests := []struct {
+		name  string
+		flags []string
+		want  map[string]string
+	}{
+		{
+			"openfaas",
+			[]string{
+				"nats.external.enabled=true",
+				"nats.external.host=openfaas-stan.openfaas.svc.cluster.local",
+				"nats.external.port=4222",
+				"nats.clusterName=\"faas-cluster\"",
+			},
+			map[string]string{
+				"nats.external.enabled": "true",
+				"nats.external.host":    "openfaas-stan.openfaas.svc.cluster.local",
+				"nats.external.port":    "4222",
+				"nats.clusterName":      "faas-cluster",
+			},
+		},
+		{
+			"issue #592",
```

Comment:
We could improve the title of this test case.

Reply:
Sure, but I was thinking about remove test case naming all together. Should I? Or should I just rename the cases?

Reply:
I kept the naming and this time they are meaningful ones!